frontend/pulse3d/src/pages/upload-form.js

```diff
@@ -562,12 +580,19 @@ export default function UploadForm() {
 
   const handleDropDownSelect = (idx) => {
     setAlertShowed(false);
-    setFiles([uploads[idx]]); // must be an array
 
-    const filenameNoExt = removeFileExt(uploads[idx].filename);
+    let nameOverride = "";
+    if (idx === -1) {
+      setFiles([]); // must be an array
+    } else {
+      const newSelection = uploads[idx];
+      setFiles([newSelection]);
+      nameOverride = removeFileExt(newSelection.filename).join(".");
+    }
+
     setAnalysisParams({
       ...analysisParams,
-      nameOverride: filenameNoExt.join("."),
+      nameOverride: nameOverride,
```

Comment:
could just be:
```
{
  ...analysisParams,
        nameOverride
}
```

Reply:
GitHub loaded this comment right as I merged, good catch though